lib/src/footer/Footer.tsx

```diff
@@ -10,7 +10,7 @@ import FooterPropsType from "./types";
 const DxcFooter = ({
   socialLinks,
   bottomLinks,
-  copyright = "",
+  copyright = "© DXC Technology 2022. All rights reserved.",
```

Comment:
We keep having the same problem as before with this approach (update every year the copyright message). My proposal: 

<img width="787" alt="Sin título" src="https://user-images.githubusercontent.com/44321109/155975929-f7c1c592-55e4-4387-aaf0-99a442027d26.png">

And as the default value for the prop don't give anyone.

Reply:
Done